modules/compliance-unapplying-kubeletconfig.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-operator-remediation.adoc
+
+[id="compliance-unapplying-kubeletconfig_{context}"]
+= Unapplying a KubeletConfig remediation
```

Comment:
You might want to consider changing _unapplying_ to something like _removing_ in a future cleanup. _Unapply_ isn't in the American Heritage or Merriam-Webster dictionary, and we don't have specific guidance saying it is okay to use in the the SSG, ISG, or repo guidelines. It is used in the repo, but only for the Compliance Operator docs.

Just something to consider, I won't hold up a merge over it.

Reply:
@michaelryanpeter  That's a very good point. I'll pass this by dev and see if they agree.